includes/class-wc-round-tax.php

```diff
@@ -0,0 +1,183 @@
+<?php
```

Comment:
Should we move it t the new PSR4 `src` directory?

Reply:
Yes, do we need to wait for #26731 to land?

Reply:
I think we could move now.